downstream/titles/controller/controller-admin-guide/docinfo.xml

```diff
@@ -1,4 +1,4 @@
-<title>Automation Controller Administration Guide</title>
+<title>Automation Controller administration guide</title>
```

Comment:
Automation Controller is not a proper noun so this should be:

Automation controller administration guide